### PR TITLE
A lot of new endpoints

### PR DESCRIPTION
**Pull request by gcouti**
Hello, 

I create a lot of new endpoints and split it in other files. Please accept this pull request. 

Thank you

### CODE REVIEW

File: issues.go [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+	url := j.BaseUrl + j.ApiPath + "/search?jql=assignee=\"" + url.QueryEscape(user) + "\"&startAt=" + strconv.Itoa(startAt) + "&maxResults=" + strconv.Itoa(maxResults)
+	contents := j.buildAndExecRequest("GET", url, nil)
+
+	var issues IssueList
+	err := json.Unmarshal(contents, &issues)
+	if err != nil {
+		fmt.Println("%s", err)
+	}
+	//
+	//    for _, issue := range issues.Issues {
```
Comment (rande) [inline issues.go]: Why this code is commented ?

File: issues_test.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
+package gojira_test
+
+import (
+    "fmt"
+    "testing"
+    "encoding/json"
+    "github.com/hekima/go-jira-client"
```
Comment (rande) [inline issues_test.go]: This is wrong, tests are failing due to this import

**Comment by rande** [comment]
Can you squash you commits ? also can you run `go fmt` or make format to respect CS ? Also can you make sure there is no useless code as comment. Thanks

File: issues.go (hunk 1 above)
Comment (gcouti) [inline issues.go]: This is not necessary, because umarshal already do that.

**Pull request closed without merging**